package.json

```diff
@@ -8,16 +8,19 @@
     "start": "next start",
     "lint": "eslint src",
     "format": "prettier --write .",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "prisma:format": "prisma format"
   },
   "dependencies": {
+    "@prisma/client": "^4.14.1",
     "@types/node": "20.2.5",
     "@types/react": "18.2.7",
     "@types/react-dom": "18.2.4",
     "eslint-config-next": "13.4.4",
     "next": "13.4.4",
     "react": "18.2.0",
-    "react-dom": "18.2.0"
+    "react-dom": "18.2.0",
+    "uuid": "^9.0.0"
```

Comment:
**nitpick:** You can remove `uuid` package, because Prisma have UUID built-in by default